keyring/backend.py

```diff
@@ -100,6 +100,11 @@ def get_password(self, service: str, username: str) -> typing.Optional[str]:
         """Get password of the username for the service"""
         return None
 
+    @abc.abstractmethod
+    def list_password(self, service: str) -> list[str]:
```

Comment:
How about name it `list_accounts` or `list_users`, since "username" and "account" are the terms used for these fields and "passwords" aren't involved in the result.